test/studies/bale/aggregation/histo.chpl

```diff
@@ -44,6 +43,5 @@ proc main() {
   }
   stopTimer("AGP");
 
-  assert(numUpdates == +reduce A.read());
```

Comment:
It's not clear to me:  Are the changes in this file related to this PR?

Reply:
This file is currently no-tested it, but I compiled/ran it manually since it uses some of the internal aggregator primitives. This revealed some bitrot in the test, like the `A` array no longer being used so I fixed that here. This test should be run as part of nightly correctness and perf testing, but it's not just reflecting that we haven't spent as much time on atomic aggregators yet.